config/config.go

```diff
@@ -368,8 +374,12 @@ func NewConfig() (c *Config) {
 	// Kubernetes client Configuration
 	c.KubernetesConfig.Burst = getDefaultInt(EnvKubernetesBurst, 200)
 	c.KubernetesConfig.QPS = getDefaultFloat32(EnvKubernetesQPS, 175)
-	c.KubernetesConfig.CacheEnabled = getDefaultBool(EnvKubernetesCacheEnabled, false)
-	c.KubernetesConfig.CacheDuration = getDefaultInt64(EnvKubernetesCacheDuration, time.Duration(5*time.Minute).Nanoseconds())
+
+	// Cache config
+	c.CacheConfig.CacheEnabled = getDefaultBool(EnvKubernetesCacheEnabled, false)
+	c.CacheConfig.CacheDuration = getDefaultInt64(EnvKubernetesCacheDuration, time.Duration(10*time.Minute).Nanoseconds())
```

Comment:
Question.
In theory client-go client caching mechanism are dealing with the TTL of the cache.
I mean, at least for the k8s objects is listening the events and it refreshes them under the hood.

In that scenario, why do we need to define a duration ?

Reply:
We give it to the client-go, which refreshes (resyncs) the cache. The 10 mins is the default setting in the client-go currently.